dids/diddht/diddht.go

```diff
@@ -19,6 +19,8 @@ import (
 	"github.com/tv42/zbase32"
 )
 
+const defaultGatewayURL = "https://diddht.tbddev.org"
```

Comment:
can we make this configurable/overridable somehow?

in practice we'll want to point this to our prod node (when it's ready...) though it should default to dev